utils/config/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+use std::io::Read;
+
+use json_comments::StripComments;
+
+pub fn strip_comments_from_json_str(json_str: &String) -> std::io::Result<String> {
```

Comment:
Good idea to move all the code that deals with `json_comments` to a separate module.
:+1: